pkg/virt-operator/resource/apply/apps_test.go

```diff
@@ -1146,20 +1146,29 @@ var _ = Describe("Apply Apps", func() {
 		})
 
 		DescribeTable("Should remove Kubevirt service accounts from the default privileged SCC", func(additionalUserlist []string) {
-			var expectedJsonPatch string
 			var serviceAccounts []string
 			saMap := rbac.GetKubevirtComponentsServiceAccounts(namespace)
 			for key := range saMap {
 				serviceAccounts = append(serviceAccounts, key)
 			}
 			serviceAccounts = append(serviceAccounts, additionalUserlist...)
 			scc := generateSCC("privileged", serviceAccounts)
+			patchSet := patch.New()
+			finalizerPath := "/users"
```

Comment:
please use `usersPath` as name or something referencing users and not finalizers